pkg/installer/generateconfig.go

```diff
@@ -318,3 +319,10 @@ func determineVMNetworkingType(vmSku *mgmtcompute.ResourceSku) string {
 	}
 	return string(vmNetworkingType)
 }
+
+func determineCredentialsMode(pwip *api.PlatformWorkloadIdentityProfile) types.CredentialsMode {
+	if pwip != nil {
```

Comment:
Although an util function will be used to determine if it is a MIWI cluster, currently we are using the safe approach to determine this:-
`oc.Properties.PlatformWorkloadIdentityProfile != nil && oc.Properties.ServicePrincipalProfile == nil`

Reply:
Understood; I'll update the logic